### PR TITLE
2022 site updates

### LINKED ISSUE #40: Inaccurate description of duplicate resources
The exhaustive configuration example says:
```
If the reference matches a generated entry, it will replace the generated entry. If it doesn't match any generated entries, it will be added to the generated entries
```
in the part about predefined resources. This is no longer the case as of this PR: https://github.com/FHIR/sushi/pull/633. So that documentation needs to be updated.

### PR DESCRIPTION
**Pull request by julianxcarter**
Makes several updates to the FSHSchool content to align things with how GoFSH and SUSHI currently work:

- Fixes #40 by adjusting the language surrounding predefined resources in the exhaustive config.
- Updates the example SUSHI box throughout the site to utilize the new layout.
- Updates the language in the GoFSH tutorial to align with the new `file-per-definition` output structure.
- Updates the list of values imported from an IG resource when using the FSHOnly option. 
- Updates the list of GoFSH CLI parameters.
- Updates SUSHI and GoFSH version numbers used in example install commands to be less... old.

### CODE REVIEW

**Review by mint-thompson — APPROVED** [review]

File: content/docs/SUSHI/configuration/_index.md [hunk 1]
```diff
@@ -50,10 +50,15 @@ SUSHI can generate a simple configuration file for you with the `--init` [option
 #### FSH-Only
 
 If an author wants SUSHI only to build the FHIR definition files, and _not_ to do any additional IG processing, and if the project contains an **ImplementationGuide** resource, then the author does not need to provide a **sushi-config.yaml** at all. If there is no **sushi-config.yaml**, SUSHI will automatically attempt to extract the following information from an **ImplementationGuide** resource:
+* `id`
 * `canonical`
+* `url`
+* `name`
+* `packageId`
```
Comment (jafeltra) [inline content/docs/SUSHI/configuration/_index.md:57]: Where are these properties derived from? I tried looking through the code for loading configuration from an IG resource, but I don't see it creating configuration properties for `id`, `url`, `name`, or `packageId`. I do see `parameters` though!

**Review by jafeltra — COMMENTED** [review]
This looks good! I just had one questions about one change!

File: content/docs/SUSHI/configuration/_index.md (hunk 1 above)
Comment (jafeltra) [reply content/docs/SUSHI/configuration/_index.md:57]: Oh wait maybe I understand. This is the list of allowed properties when you use `FSHOnly: true` right? If so, I think the sentence before the list is misleading. That sentence says that this list is the properties extracted from an IG resource, not ones that `sushi-config.yaml` supports. If this is the case, then I think it makes sense to remove `id`, `url`, `name`, and `packageId` from this list, and maybe include a separate list or description about what can be included in a `FSHOnly` `sushi-config.yaml` file.

Comment (julianxcarter) [reply content/docs/SUSHI/configuration/_index.md:57]: I got these parameters from [the `loadConfigurationFromIgResource` function](https://github.com/FHIR/sushi/blob/888b0c70e57f9a8b0180036ff0e614bb6b6f7297/src/import/loadConfigurationFromIgResource.ts#L62), I think you may have been looking at an older branch? `id`, `url`, `name` and `packageId` were all added fairly recently!

Comment (jafeltra) [reply content/docs/SUSHI/configuration/_index.md:57]: Oh.. the irony is that I was on `next`. My bad!

**Review by jafeltra — APPROVED** [review]

**Review by cmoesel — APPROVED** [review]
This all looks good to me!

**Pull request merged**